src/gui/accountsettings.cpp

```diff
@@ -333,6 +334,47 @@ void AccountSettings::slotEditCurrentIgnoredFiles()
     openIgnoredFilesDialog(f->path());
 }
 
+void AccountSettings::slotOpenMakeFolderDialog()
+{
+    const auto &selected = _ui->_folderList->selectionModel()->currentIndex();
+
+    if (!selected.isValid()) {
+        qCWarning(lcAccountSettings) << "Selection model current folder index is not valid.";
+        return;
+    }
+
+    const auto &classification = _model->classify(selected);
+
+    if (classification != FolderStatusModel::SubFolder && classification != FolderStatusModel::RootFolder) {
+        return;
+    }
+
+    const QString fileName = [this, &selected, &classification] {
+        QString result;
+        if (classification == FolderStatusModel::RootFolder) {
+            const auto alias = _model->data(selected, FolderStatusDelegate::FolderAliasRole).toString();
+            const auto folderMan = FolderMan::instance();
+            if (const QPointer<Folder> folder = folderMan->folder(alias)) {
```

Comment:
Either this is back or I missed the fact that the comment wasn't addressed but you don't need a QPointer here, a raw pointer is enough. There's no chance that object will be destroyed between the `folder` and `path` calls.

Reply:
@er-vin I guess you missed it. Alright. This was copy-pasted from another method :(

Reply:
@er-vin Was blindly copy-pasted from `void AccountSettings::slotCustomContextMenuRequested(const QPoint &pos)`

Reply:
FYI I guess it made sense there at some point in time if one of slots connected to the actions were receiving that pointer, but not anymore since none of them use it (the folder is refetched in the slots anyway).

Reply:
@er-vin Ok. Removed it. And, accidentally, cleared the history. But you can locate this change I am sure. This was the only one.